vmware_exporter/vmware_exporter.py

```diff
@@ -873,27 +891,26 @@ class HealthzResource(Resource):
     def render_GET(self, request):
         request.setHeader("Content-Type", "text/plain; charset=UTF-8")
         request.setResponseCode(200)
-        log("Service is UP")
+        logging.info("Service is UP")
         return 'Server is UP'.encode()
 
-
-def log(data, *args):
-    """
-    Log any message in a uniform format
-    """
-    print("[{0}] {1}".format(datetime.datetime.utcnow().replace(tzinfo=pytz.utc), data % args))
-
-
 def main(argv=None):
     """ start up twisted reactor """
     parser = argparse.ArgumentParser(description='VMWare metrics exporter for Prometheus')
     parser.add_argument('-c', '--config', dest='config_file',
                         default=None, help="configuration file")
     parser.add_argument('-p', '--port', dest='port', type=int,
                         default=9272, help="HTTP port to expose metrics")
+    parser.add_argument('-l', '--loglevel', dest='loglevel',
```

Comment:
I might of missed this, but can you do a env variable for this as well?

Reply:
Preferably in another change/PR, okay?

Reply:
Works for me.